pubspec.yaml

```
@@ -15,34 +15,48 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 1.0.0+1
+version: 0.2.0+1
```

oh, so the first version is `1.0.0`?? 😅 btw, you have to increase the build number as well
```suggestion
version: 0.2.0+2
```

---

@manh-t the build number will be automatically increased by build runner
- https://github.com/luongvo/flutter-survey/pull/54/files#diff-d51300f9a04aa847194d5d92e6d258e82e93953bfc60ff8aacc97dae0cd24a51R38
- https://github.com/luongvo/flutter-survey/pull/54/files#diff-d51300f9a04aa847194d5d92e6d258e82e93953bfc60ff8aacc97dae0cd24a51R75

---

ah I got it ⚡